### PR TITLE
Added new 2020-06-01 api version for Azure Networking

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
open api link: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/network/resource-manager/Microsoft.Network/stable/2020-06-01

### CODE REVIEW

**Comment by MikhailTryakhov** [comment]
@allenjzhang  can you please approve it asap?

**Comment by allenjzhang** [comment]
Could you:

 - Add a link to the openapi spec review PR in initial PR comments
 - Update api tag in `AzSdk.RP.props`,  and add release notes in csproj 
 
Thanks

**Comment by allenjzhang** [comment]
Also in the future, if you can mark with label `mgmt` that would be great

**Comment by MikhailTryakhov** [comment]
@allenjzhang done

File: sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -10,11 +10,13 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Network" (https://www.nuget.org/packages/Azure.ResourceManager.Network/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>20.0.4-preview</Version>
+		<Version>20.1.0-preview</Version>
```
Comment (allenjzhang) [inline sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj:13]: I see the swagger is in stable version now, do you still need -preview?

Comment (MikhailTryakhov) [reply sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj:13]: No, can I simply call it 20.1.0?

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**